path_config.go

```diff
@@ -23,6 +26,15 @@ func pathConfig(b *versionedKVBackend) *framework.Path {
 				Type:        framework.TypeBool,
 				Description: "If true, the backend will require the cas parameter to be set for each write",
 			},
+			"delete_version_after": {
```

Comment:
We shouldn't do this. People are not using to giving versions as strings especially as API users or curl users. We need to figure out another way.